aten/src/ATen/native/native_functions.yaml

```diff
@@ -6359,7 +6384,7 @@
   variants: function, method
   dispatch:
     CompositeExplicitAutograd: unbind
-    NestedTensorCPU, NestedTensorCUDA: NestedTensor_unbind
+    AutogradNestedTensor, NestedTensorCPU, NestedTensorCUDA: NestedTensor_unbind
```

Comment:
nit: can you add a comment saying that this makes it CompositeImplicit-like for nested Tensors!